### DISCUSSION TITLE
Would KEDA be open to doing patch updates after Dependabot commits?

### DISCUSSION THREAD

**Discussion by qpetraroia** [kedacore/keda#5909] [Ideas]
I noticed that Dependabot is enabled on the repository. It would be beneficial to have more patch releases that cover Dependabot updates. Would the community be open to more releases addressing these dependency updates? I believe this would be a great way to release patch updates that cover vulnerabilities that Dependabot finds.

**Comment by zroubalik** [comment]
Unfortunately we don't have a bandwith to do more releases off-cycle, we usually od 0-2 off-cycle release if there are some critical bugs in the codebase.

**Reply by qpetraroia** [reply]
Is there usually a timeline between releases? Does KEDA have a CVE policy as well? Thanks @zroubalik

**Reply by zroubalik** [reply]
Please see our roadmap for details: https://github.com/kedacore/keda/blob/main/ROADMAP.md

**Reply by davogen** [reply]
I am also interested in this.

**Reply by davogen** [reply]
@qpetraroia FYI I have found this on the KEDA website on the Support page: https://keda.sh/support/#commercial-support

I wonder if these vendors provide patch releases. We have a strict CVE and security policy at our company, so this might be a solution.
